pennylane/transforms/dynamic_one_shot.py

```diff
@@ -87,31 +87,63 @@ def func(x, y):
     for m in tape.measurements:
         if not isinstance(m, (CountsMP, ExpectationMP, ProbabilityMP, SampleMP, VarianceMP)):
             raise TypeError(
-                f"Native mid-circuit measurement mode does not support {type(m).__name__} measurements."
+                f"Native mid-circuit measurement mode does not support {type(m).__name__} "
```

Comment:
What measurement processes are we not supporting here?

Reply:
Ah okay for some reason I thought those measurement classes cover more measurements than just those 😅